code/modules/admin/verbs/plane_debugger.dm

```diff
@@ -63,8 +63,12 @@
 
 /datum/plane_master_debug/proc/get_target()
 	var/mob/target = mob_ref?.resolve()
-	if(!target?.hud_used)
+	if(!target?.hud_used || target == owner.owner.mob)
 		target = owner.owner.mob
+		if (ismob(target.orbit_target)) // If we're orbiting someone, swap to them if possible
```

Comment:
I don't think I really like the behavior, focusing a mob should always be intentional, not a side effect (if I'm reading this right)

Reply:
What is the situation in which this isn't desirable behavior? If you enable the focus mode (which displays that its in orbit mode with the icon), you expect to focus on someone, not yourself?